lib/notifier.js

```diff
@@ -24,7 +24,7 @@ if (notifierSnsTopicArn !== 'disabled') {
   // This will pull in default credentials, region data etc
   // from the metadata service available to the instance.
   // It's magic, and it's awesome.
-  sns = new AWS.SNS({region: region})
+  sns = new AWS.SNS({endpoint: 'http://localhost:4100/', region: region})
```

Comment:
@deeptibaghel to merge this let's make the `endpoint` configurable. 

Add a new block into https://github.com/mozilla/fxa-auth-server/blob/master/config/index.js#L442 before `snsTopicArn`, we can call this `snsTopicEndpoint` for now. default will be `undefined`. and `http://localhost:4100/` will be set in `dev.json`

Reply:
sorry `snsTopicEndpoint` will be listed after `snsTopicArn` in the config